test/lib/model/test_yake_keywords.py

```diff
@@ -32,6 +32,17 @@ def test_run_yake(self, mock_yake_response):
         mock_yake_response.return_value = [["ball", 0.23]]
         self.assertEqual(self.yake_model.run_yake(**self.yake_model.get_params(message)), {"keywords": [["ball", 0.23]]})
 
+    def test_run_yake_real(self):
+        message = schemas.Message(**{
+            "body": {
+                "id": "1234",
+                "text": "I love Meedan",
+            },
+            "model_name": "yake_keywords__Model"
+        })
+        results = self.yake_model.run_yake(**self.yake_model.get_params(message))
+        self.assertEqual(results, {"keywords": [('love Meedan', 0.0013670273525686505)]})
```

Comment:
Nice! looks like we submitted an id, but it didn't come back out of the model?  Or is presto going to handle the linking of request id payload to the keywords returned?
Also, can we use a more complex realistic example where we will get multiple keywords back?

Reply:
`process` function in `yake_keywords.py` returns GenericType which contains the id as well. This test case is for `run_yake`
I'll commit a longer test case